Quadriga/src/main/webapp/WEB-INF/views/transformation/statusTransformation.jsp

```diff
@@ -0,0 +1,6 @@
+<META HTTP-EQUIV="refresh" CONTENT="5;URL=${pageContext.servletContext.contextPath}/auth/transformation/result/status">
+
+<h1> Transformation is running... </h1>
+
+<p>The resulting graph of the transformation is being generated.
```

Comment:
The transformation is running. Please stay on this page. It will display a download link, once the transformation has finished.